### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by sfc-gh-ghernandez**

### CODE REVIEW

File: site/sfguides/src/dora/snowflake-auto-grader.md [hunk 1]
```diff
@@ -75,15 +75,14 @@ create or replace external function util_db.public.grader(
  , description varchar) 
  returns variant 
  api_integration = dora_api_integration 
- context_headers = (current_timestamp,current_account, current_statement) 
+ context_headers = (current_timestamp,current_account, current_statement, current_account_name)
```
Comment (sfc-gh-ghernandez) [inline site/sfguides/src/dora/snowflake-auto-grader.md:78]: Adding `current_account_name`

File: site/sfguides/src/dora/snowflake-auto-grader.md [hunk 2]
```diff
@@ -99,16 +98,69 @@ select grader(step, (actual = expected), actual, expected, description) as grade
  'DORA_IS_WORKING' as step
```
Comment (sfc-gh-ghernandez) [inline site/sfguides/src/dora/snowflake-auto-grader.md]: Does this need to change too? The name of the step here

File: site/sfguides/src/dora/snowflake-auto-grader.md [hunk 3]
```diff
@@ -99,16 +98,69 @@ select grader(step, (actual = expected), actual, expected, description) as grade
...
 );
 ```
 
-If the auto-grader is correctly provisioned, you should see a **GRADED_RESULTS** column with several pieces of information, including a checkbox and a message `"description": "Dora is working!"`.
+If the auto-grader is correctly provisioned, you should see a **GRADED_RESULTS** column with several pieces of information, including a checkbox and a message `"description": "The Snowflake auto-grader has been successfully set up in your account!"`.
+<!-- ------------------------ -->
+## Create the `greeting` function
+Duration: 2
+
+Next, you'll create the `greeting` function. This function will link your Snowflake account to our badging system. If you don't complete this step, you'll risk not receiving your badge.
```
Comment (sfc-gh-ghernandez) [inline site/sfguides/src/dora/snowflake-auto-grader.md:110]: Is this accurate?

File: site/sfguides/src/dora/snowflake-auto-grader.md [hunk 4]
```diff
@@ -99,16 +98,69 @@ select grader(step, (actual = expected), actual, expected, description) as grade
...
+
+```sql
+use role accountadmin;
+use database util_db;
+
+-- Replace <email> with your email address
+-- Replace <first name> <middle name> <last name> with your names
+select util_db.public.greeting(<email>, <first name>, <middle name>, <last name>);
+
+-- select util_db.public.greeting('my@email.com', 'Snowflake', 'The', 'Bear');
```
Comment (sfc-gh-ghernandez) [inline site/sfguides/src/dora/snowflake-auto-grader.md:156]: What do you think about this example?

File: site/sfguides/src/dora/snowflake-auto-grader.md (hunk 3 above)
Comment (sfc-gh-ktyler) [reply site/sfguides/src/dora/snowflake-auto-grader.md:110]: yes

File: site/sfguides/src/dora/snowflake-auto-grader.md (hunk 4 above)
Comment (sfc-gh-ktyler) [reply site/sfguides/src/dora/snowflake-auto-grader.md:156]: that's cute.

File: site/sfguides/src/dora/snowflake-auto-grader.md (hunk 2 above)
Comment (sfc-gh-ktyler) [reply site/sfguides/src/dora/snowflake-auto-grader.md]: yes

File: site/sfguides/src/dora/snowflake-auto-grader.md [hunk 5]
```diff
@@ -96,19 +95,72 @@ use database util_db;
 use schema public;
 
 select grader(step, (actual = expected), actual, expected, description) as graded_results from (SELECT
- 'DORA_IS_WORKING' as step
+ 'AUTO_GRADER_IS_WORKING' as step
```
Comment (sfc-gh-ghernandez) [inline site/sfguides/src/dora/snowflake-auto-grader.md:98]: Here's that update

**Review by sfc-gh-ktyler — APPROVED** [review]

**Pull request merged**